c++/measures/g4_iw.cpp

```diff
@@ -106,6 +109,8 @@ namespace cthyb {
     }
   }
 
+  // Define placeholders once and for all in types.hpp?
```

Comment:
See a0d6b5a3e4fcd2893a4ba3a7926b9d880f5dddb5

One question. Why do you do the allocation/deallocation of fftw manually in your nfft_buf?